src/symbols/Node.tsx

```diff
@@ -20,12 +20,11 @@ import {
   CollapseProps
 } from '../types';
 import { Html, useCursor } from '@react-three/drei';
-import { useCameraControls } from '../CameraControls';
+import { useCameraControls } from '../hooks';
```

Comment:
Lets avoid barrels like this. It should import from the specific file for better optimization.